### ISSUE TITLE
Using Relative Links in Blog Post Text

### ISSUE THREAD

**Issue by schallerstudios** [rainlab/blog-plugin#440]
Hello,
Thank you for your work on the Blog Plugin. It's been very valuable for us. :)

My question is, how do I include relative links in a blog post?
When I try, the blog rendering engine assumes that the link relates to another blog post and does not allow me to make it relative to the site root.

I have not been to find an answer in the documentation. I did try to use Twig tags, but that rendered as text. And though I might have missed it, the Markdown syntax tool didn't reveal any options that seemed to fit either.

Basically, I want links within a blog post to interface with static pages or other plugin content. I could use absolute links in the posts, but I would rather figure this out, so that when I am done with development, I can copy the finished site to the live server without mountains of editing and the risk of broken links.

Thanks! :)

**Comment by LukeTowers** [comment]
Same as you would any other way? Just start the link with `/` and that will make it relative to the domain root.

**Comment by schallerstudios** [comment]
The problem is that in my development environment, October is installed in a subfolder, not at the top level, whereas the live site is installed at the top level. The url('/') method delivers the correct root location for the October install so that links work both in the development environment and the production environment. But I haven't found an equivalent that works in the blog post copy.

That's what I meant by not having to edit every blog post when I migrate the site or each time I make a major edit to the site. Currently, to follow the method you suggested, I must manually enter the subfolder names into each link.

So, Is this the only way, or is there a method like link('/') that will work in a blog post?

Note: I don't have the ability to install October on the top level. I do not have the access to create multiple VMs and I am developing other sites on the same server.

**Comment by LukeTowers** [comment]
The blog post copy is only run through a markdown parser, so there is nothing built in that would provide what you're looking for. You could however register a custom Twig filter in your plugin (https://octobercms.com/docs/plugin/registration#extending-twig) that you could use to replace a preset string of your choice with the root url dynamically when rending the blog post content.

Something like
```markdown
[link here](ROOT_URL/link-to-post)
```
```twig
{{ post.content_html | mycustomfilter | raw }}
```
